src/common/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import cn from 'classnames'
+
+export type TBreadcrumb = {
+  label: string
+  route?: string
+  className?: string
+}
+
+export type TBreadcrumbs = {
+  breadcrumbs: TBreadcrumb[]
+  className?: string
+}
+
+const Breadcrumbs = (props: TBreadcrumbs): JSX.Element => {
+  const { breadcrumbs, className } = props
+
+  return (
+    <div
+      className={cn(
+        'wrapper py-0 flex font-normal text-14px leading-19px text-gray-71 mb-12px pt-12px',
+        className,
+      )}
+    >
+      {breadcrumbs.map((breadcrumb, index) => {
+        if (breadcrumb.route) {
+          return (
+            <React.Fragment key={index}>
+              <a href={breadcrumb.route}>{breadcrumb.label}</a>
```

Comment:
Please use Link from react-router-dom instead of a tag

Reply:
fixed